packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -32,12 +33,15 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
         profit,
         date_start,
         tick_count,
+        tick_passed,
         transaction_ids: { buy, sell } = {},
     } = contract_info;
 
     const is_profit = profit >= 0;
-
     const cancellation_price = getCancellationPrice(contract_info);
+    const ticks_duration_text = isAccumulatorContract(contract_type)
+        ? `${tick_passed}/${tick_count} ${localize('ticks')}`
```

Comment:
Accumulator contracts doesn't support 1 tick?

Reply:
@balakrishna-binary it does but not for the tick_count. there is no case like that where we would have 1/1 tick (1 out of 1 tick) 😄 
tick_count is always more than 1 tick.